File: c_sharp/SystemUnderTest/Contacts.cs
namespace SystemUnderTest {
    public class Contacts {
        public class Info {
            public string FirstName { get;  set; }
            public string LastName { get;  set; }
            public string GitHubUsername { get;  set; }
        }

        public static Info get(string name) {
            Info info = new Info() { FirstName = "Paul",
                LastName = "Moore", 
                GitHubUsername = "pdmoore"
            };

            return info;
        }
    }
}

File: junit_5.6.2/src/main/java/com/rocketnine/junitexamples/Contacts.java
package com.rocketnine.junitexamples;

public class Contacts {

    public static class Info {
        private final String firstName;
        private final String lastName;
        private final String githubUsername;

        public Info(String firstName, String lastName, String githubUsername) {
            this.firstName = firstName;
            this.lastName  = lastName;
            this.githubUsername = githubUsername;
        }

        public String getFirstName() {
            return firstName;
        }

        public String getLastName() {
            return lastName;
        }

        public String getGithubUsername() {
            return githubUsername;
        }
    }

    public static Info get(String name) {
        return new Info("Paul", "Moore", "pdmoore");
    }
}
